### DISCUSSION TITLE
How to write a byte[] to a redis list

### DISCUSSION THREAD

**Discussion by psasikaladevi** [redis/lettuce#2086] [Q&A]
Hi, we have a scenario where we need to write a byte[] to a redis list. Can we get some help in there on how to achieve it using lettuce?

**Comment by mp911de** [comment]
All values (list entries, fields in a hash, …) in Redis are binary. If you need binary representation of values, take a look at `ByteArrayCodec`. Depending on what you're trying to achieve, you can build a `RedisCodec<String, byte[]>` that delegates key encoding to `StringCodec` and value operations to `ByteArrayCodec`.
